docs/backup/03-01-install-velero.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Install Velero
+type: Details
+---
+Install and configure [Velero](https://github.com/heptio/velero/) to be able to back up and restore your Kyma cluster.
+
+## Velero setup
+
+To successfully set up Velero, provide a supported storage location and credentials to access it. Currently, you can install Velero on GCP and Azure. AWS is not supported.
+
+Follow the instructions below:
+1. Enable the installation of the velero components on the kyma installer. To do that follow [this guide](/root/kyma/#configuration-custom-component-installation).<br/>
+    The components to add are:
+    ```yaml
+    - name: "velero-essentials"
+      namespace: "kyma-backup"
+    - name: "velero"
+      namespace: "kyma-backup"
+    ```
+
+2. Create an override secret containing the velero [required parameters](/components/backup/#configuration-configuration) for the provider of your choosing **encoded in base 64**.<br/>
+    See the examples below for reference (_values are in plain text for illustrative purposes, remember to set them as base64 encoded strings_):
+
+    <div tabs>
+    <details>
+    <summary>
+    Google Cloud Platform
+    </summary>
+        
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+    name: velero-credentials-overrides
+    namespace: kyma-installer
+    labels:
+        kyma-project.io/installation: ""
+        installer: overrides
+        component: velero
+    type: Opaque
+    data:
+        configuration.provider: "gcp"
+        configuration.volumeSnapshotLocation.name: "gcp"
+        configuration.volumeSnapshotLocation.bucket: "my-gcp-bucket"
+        configuration.backupStorageLocation.name: "gcp"
+        configuration.backupStorageLocation.bucket: "my-gcp-bucket"
+        credentials.secretContents.cloud: |
+                    {
+                        "type": "service_account",
+                        "project_id": "my-project",
+                        "private_key_id": "KEY_UUID",
+                        "private_key": "-----BEGIN PRIVATE KEY-----\nPRIVATE_KEY_CONTENTS\n-----END PRIVATE KEY-----\n",
+                        "client_email": "sample@fake.iam.gserviceaccount.com",
+                        "client_id": "MY_CLIENT_ID",
+                        "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+                        "token_uri": "https://oauth2.googleapis.com/token",
+                        "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+                        "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/sample%40fake.iam.gserviceaccount.com"
+                    }
+    ```
+    >**NOTE:** For details on configuring and installing Velero in GCP,  see [this](https://velero.io/docs/v1.0.0/gcp-config/) document.
+    </details>
+    <details>
+    <summary>
+    Azure
+    </summary>
+
+    Coming soon...
```

Comment:
When? :D Just a question - will we have something here for the release?

Reply:
Yes, I will work on this right after finishing this docs.